### PR TITLE
Supporty exporting query for sample facet selection

### PR DESCRIPTION
**Pull request by bfcrampton**

### CODE REVIEW

**Comment by bfcrampton** [comment]
FYI @mbookman

File: api/data_explorer/controllers/export_url_controller.py [hunk 1]
```diff
@@ -165,32 +165,58 @@ def _get_range_clause(column, value):
...
+    return split[0], split[1], is_sample_file_column
+
+
 def _get_filter_query(filters):
+    sample_file_column_fields = {
+        k.lower().replace(" ", "_"): v
+        for k, v in current_app.config['SAMPLE_FILE_COLUMNS'].iteritems()
+    }
+
+    current_app.logger.info('FILTERSSSS: %s' % filters)
```
Comment (melissachang) [inline api/data_explorer/controllers/export_url_controller.py]: Delete?

File: api/data_explorer/controllers/export_url_controller.py [hunk 2]
```diff
@@ -165,32 +165,58 @@ def _get_range_clause(column, value):
...
+
+def _get_table_name_column(es_field_name, sample_file_column_fields):
+    is_sample_file_column = False
+    if es_field_name.startswith('samples.'):
+        es_field_name = es_field_name.replace('samples.', '')
+        # Check if this is one of the special '_has_<file_type>' facets.
+        stripped = es_field_name.replace('_has_', '')
+        if stripped in sample_file_column_fields:
+            es_field_name = sample_file_column_fields[stripped]
+            is_sample_file_column = True
```
Comment (melissachang) [inline api/data_explorer/controllers/export_url_controller.py]: `is_sample_file_column` sounds like "is one of the columns in bigquery.json sample_file_columns, eg bryancrampton-testing.data_explorer.1000_genomes_sample_info.chr1_vcf

How about `is_has_sample_file_type_field`

File: api/data_explorer/controllers/export_url_controller.py (hunk 1 above)
Comment (bfcrampton) [reply api/data_explorer/controllers/export_url_controller.py]: Oops 😄

File: api/data_explorer/controllers/export_url_controller.py (hunk 2 above)
Comment (bfcrampton) [reply api/data_explorer/controllers/export_url_controller.py]: Done (`has_sample_file_type_field`)

**Pull request merged**